overview.html

```diff
@@ -41,7 +41,7 @@ <h1>SpatioTemporal Asset Catalogs In Depth</h1>
                         	<p>There are exabytes of spatial data in existence today, with the potential to greatly increase our understanding of the world and improve decisions. But most of it remains inaccessible, as it is in a myriad of different formats, with no common way to find relevant information about a place over time. As data migrates to the cloud, we have an opportunity to make it more accessible and interoperable. STAC is a standard and a community of collaborators working to enable increased access to information about our planet. You can learn more about the motivations for the standard on the <a href="why-stac.html">Why STAC?</a> page.</p>
 							<p>The philosophy of STAC is to focus on making it as easy as possible for data providers to expose their data to the world. Most geospatial catalogs require the provider of data to maintain servers and databases to enable search. This can be a large challenge with huge amounts of data. STAC aims to flip that paradigm on its head, following the path of the web, where putting up an HTML page is very simple, and then the search of those pages can be done by anyone, with experts in search emerging over time. STAC aims to enable that next generation of geospatial search engines, while also supporting web best practices so geospatial information is more easily surfaced in traditional search engines.</p>
-							<p>The STAC spec itself provides a lowest common denominator JSON format to wrap around any relevant data about the earth. The core <a href="http://geojson.org">GeoJSON</a> object and related structures are designed for extension, so it can adapt to different domains. For more information about the specification, see the <a href="core.html">Spec Overview</a> page. The core JSON pages can be transformed into browsable, interactive HTML pages with tools like <a href="https://github.com/radiantearth/stac-browser/">STAC Browser</a>. When STAC is used in concert with emerging formats like <a href="http://cogeo.org"</a>Cloud Optimized GeoTIFF</a> the result vastly lowers the barriers for anybody to find and use geospatial assets like satellite imagery:</p>
+							<p>The STAC spec itself provides a lowest common denominator JSON format to wrap around any relevant data about the earth. The core <a href="http://geojson.org">GeoJSON</a> object and related structures are designed for extension, so it can adapt to different domains. For more information about the specification, see the <a href="core.html">Spec Overview</a> page. The core JSON pages can be transformed into browsable, interactive HTML pages with tools like <a href="https://github.com/radiantearth/stac-browser/">STAC Browser</a>. When STAC is used in concert with emerging formats like <a href="http://cogeo.org"</a>Cloud Optimized GeoTIFF</a> or <a href="https://github.com/TileDB-Inc/TileDB/">TileDB</a></href>the result vastly lowers the barriers for anybody to find and use geospatial assets like satellite imagery:</p>
```

Comment:
That `</href>` is strange...

Reply:
It is, I will remove that. I noticed that the anchor for COG is also odd, I will edit that one as well.